crates/services/sync/src/import/tests.rs

```diff
@@ -14,7 +14,11 @@ use crate::{
     },
```

Comment:
It would be nice to see test where execution fails, and we see that we will not call `p2p` because all data is fetched already.

Reply:
I made a test locally that is failing because this is not the behavior we decided together. I think we said that if execution fails we should remove the data from the cache because it would probably fails again. The line that clears the cache : https://github.com/FuelLabs/fuel-core/blob/a566ac569581447cbcc24cc4677bdb84f9c1c9be/crates/services/sync/src/import.rs#L340